starknet_py/net/account/account_client.py

```diff
@@ -73,6 +75,34 @@ async def _get_nonce(self) -> int:
         )
         return nonce
 
+    async def get_balance(self, token_address: AddressRepresentation = None) -> int:
+        """
+        Checks account's balance of specified token._
```

Comment:
```suggestion
        Checks account's balance of specified token.
```
Or is it on purpose?